Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoadTestDataPatientBean.java

```diff
@@ -56,16 +61,41 @@
 public class LoadTestDataPatientBean {
```

Comment:
This Bean has to be renamed to LoadTestDataBean as it deals with Address,Identifier etc.

Reply:
do not know if we wanted too many bean for one interface of Patient